modules/codegen/src/main/scala/com/twilio/guardrail/Error.scala

```diff
@@ -8,6 +8,8 @@ case class UnparseableArgument(name: String, message: String) extends Error
 case object NoArgsSpecified                                   extends Error
 case object NoFramework                                       extends Error
 case object PrintHelp                                         extends Error
+case class RuntimeException(message: String)                  extends Error
```

Comment:
This naming is maybe kinda confusing, no?

Reply:
Yeah, maybe `RuntimeFailure` or something